bin/lerna.js

```diff
@@ -20,7 +20,8 @@ var cli = meow([
   "Options:",
   "  --independent, -i  Version packages independently",
   "  --canary, -c       Publish packages after every successful merge using the sha as part of the tag",
-  "  --skip-git         Skip commiting, tagging, and pushing git changes (only affects publish)"
+  "  --skip-git         Skip commiting, tagging, and pushing git changes (only affects publish)",
+  "  --force-version    Force a new version for the specified packages (comma-seperated) or all packages using *"
```

Comment:
I guess we can update this too - Force a new publish.

Could also explain that it bypasses the updated/git diff check if that's helpful